### DISCUSSION TITLE
Video DownloadHelper 8.2.0.24   bug depuis mise à jour hier

### DISCUSSION THREAD

**Discussion by milsabords1** [aclap-dev/video-downloadhelper#156] [Help and Q&A]
### Your question

Ta question

Bonjour à tous
Depuis que j'ai mis à jour DownloadHelper 7 vers la version Video DownloadHelper 8.2.0.24, il ne télécharge plus les fichiers vidéo comme avant. Auparavant, il affichait toujours le titre et le téléchargeait automatiquement au format MP4. pourquoi plus ?? quelqu'un peut-il m'aider s'il vous plaît ?

**Comment by GrampaWildWilly** [comment]
La section Addon & CoApp version and details est là pour une raison.  C'est important de montrer ces détails.  Veuiller suivre ces directions: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help.  De plus, ce forum est beaucoup meilleur que l'ancien chez Google.  On peut modifier ses propres commentaires.  Chercher les 3 points en haut et à droite du commentaire.  Cocher cela.  J'imagine que ce sera évident quoi faire par la suite.
